UiPharmecy/.idea/.name

```diff
@@ -0,0 +1 @@
+UiPharmecy
```

Comment:
Read [this](https://github.com/AhsanBracu/Image-To-Pdf/pull/1/files#diff-a084b794bc0759e7a6b77810e01874f2R42) instruction: same instruction goes here.